csharp/ql/src/semmle/code/csharp/ir/internal/OperandTag.qll

```diff
@@ -0,0 +1,368 @@
+import csharp
+
+private int getMaxCallArgIndex() {
+  result = max(int argIndex |
+    exists(Call call |
+      exists(call.getArgument(argIndex))
+    ) or
+    // Quick fix so that generated calls (`Invoke` etc) will have the
+    // correct number of parameters; it is an overestimation,
+    // since we don't care about all the callables, so it
+    // should be restricted more
+    exists(Callable callable |
+      callable.getNumberOfParameters() = argIndex
+    )
+  )
+}
+
+private newtype TOperandTag =
```

Comment:
This seems like a lot of new code, which I thought was already shared with C++?

Reply:
All `internal` folders are not shared with the C++ IR.

Reply:
Looks like you accidentally made a copy then? The file already exists at `ir/implementation/internal/OperandTag.qll`

Reply:
Sorry about that, `OperandTag.qll` was indeed shared and missed this somehow. I've deleted the file `ir/internal/OperandTag.qll`.